prep-html-syntax/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Oops! You are missing your `head` element. It's a very understandable thing to miss since there's no content for it in this exercise, but it is still a mandatory part of any HTML document 😁